src/coreclr/src/tools/r2rtest/Commands/CompileNugetCommand.cs

```diff
@@ -80,7 +80,7 @@ public static int CompileNuget(BuildOptions options)
                     }
 
                     // This is not a reliable way of building the publish folder
-                    string publishFolder = Path.Combine(appFolder, @"artifacts\Debug\net5.0\publish");
+                    string publishFolder = Path.Combine(appFolder, @"artifacts\Debug\net6.0\publish");
```

Comment:
Instead of changing this over and over again, can we just pass the moniker in as a constant?